changelogs/unreleased/gh-3682-uri-encode-decode.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/uri
+
+* Added functions for encoding and decoding URL string (gh-3682).
```

Comment:
1. Our topic here is `URI`, it is more general than `URL`.
1. 'URL string' has vague meaning, I would say 'URI parts such as query parameter keys and values'.